features/api-token.feature

```diff
@@ -35,9 +35,8 @@ Feature: User API Token
         Then their token will have that new label
         And the token's 'last used' property will not be updated
 
-    @ignore
     Scenario: Revoke API Token
-        And owns an existing API token
-        When "calvin" revokes the token
+        Given "calvin" owns an existing API token named "tiger"
+        When they revoke the token
```

Comment:
Wonder why it's changed from `calvin` to `they`? Shouldn't only calvin has the ability to revoke his own token?